msg/MotionReadyEnum.msg

```diff
@@ -61,4 +61,7 @@ string NOT_READY_OTHER_TRAJ_MODE_ACTIVE_STR="Another trajectory mode is already
 uint16 NOT_READY_NOT_CONT_CYCLE_MODE=115
 string NOT_READY_NOT_CONT_CYCLE_MODE_STR="Continuous cycle mode not set. Please set the cycle mode to continuous"
 
+uint16 NOT_READY_MAJOR_ALARM=116
+string NOT_READY_MAJOR_ALARM_STR="The controller has an active Major Alarm that cannot be reset. Please check the teach pendant"
```

Comment:
```suggestion
string NOT_READY_MAJOR_ALARM_STR="Major Alarms can't be reset remotely. Please check the teach pendant"
```

Reply:
That suggested message is good as a message for if `/reset_error` is called, but as a typical `MotionNotReadyCode` string it doesn't exactly do its job of telling the user what is preventing motion. For example if `Ros_Controller_GetNotReadySubcode()` is called during `Ros_MotionControl_StartMotionMode()`, the corresponding message doesn't really fit.

Reply:
Ok, what about:

```suggestion
string NOT_READY_MAJOR_ALARM_STR="Major Alarms can't be reset (remotely). Please check the teach pendant"
```

Reply:
I just think that if I call `/start_traj_mode` and I get that message in response, it is a little confusing